src/kvstore/RocksEngine.cpp

```diff
@@ -449,9 +449,10 @@ ResultCode RocksEngine::createCheckpoint(const std::string& name) {
     auto checkpointPath = folly::stringPrintf("%s/checkpoints/%s/data",
                                               dataPath_.c_str(), name.c_str());
     LOG(INFO) << "Target checkpoint path : " << checkpointPath;
-    if (fs::FileUtils::exist(checkpointPath)) {
-        LOG(ERROR) << "The snapshot file already exists: " << checkpointPath;
-        return ResultCode::ERR_CHECKPOINT_ERROR;
+    if (fs::FileUtils::exist(checkpointPath) &&
```

Comment:
Why not return success?

Reply:
> Why not return success?

Because we don't have a way to verify the validity of the snapshot that has been created. For data security, I need to delete and recreate it at here.

Reply:
That's fine.